src/main/webapp/WEB-INF/views/series/info.html

```diff
@@ -947,18 +990,20 @@ <h5 th:text="#{t_add_info_who_selling_series}">Add info about selling/buying thi
 				}
 			};
 			+]*/
-			
+
 			/*[- */
 			var addReleaseYearProps = {
 				'url': '/series/100',
-				'l10n': {}
+				'l10n': {},
+				"sinceYear": 1840,
```

Comment:
Please, use single quotes:
```suggestion
				'sinceYear': 1840,
```